frontend/src/components/SearchTable.vue

```diff
@@ -497,8 +497,8 @@ export default {
       // store choice only once in a dict
       Object.keys(this.searchResults).forEach((componentType) => {
         const compoList = this.searchResults[componentType];
-        // sort
-        compoList.sort((a, b) => this.sortResults(a, b, this.searchedTerm));
+        compoList.sort((a, b) => sortResultsScore(a, b));
```

Comment:
If the results are already sorted by score when the API response arrives in `frontend/src/store/modules/search.js `, then perhaps it's not needed here.

Reply:
They are not sorted when arriving. But the sorting could ofc be done in the backend instead

Reply:
Oh I must have gotten confused then by this line https://github.com/MetabolicAtlas/MetabolicAtlas/blob/555e7d24a97bdfdc19be5c5382c8ab5ca25e855e/frontend/src/store/modules/search.js#L57